From df90b394c6116b7f9b71d7f4e9c42131337a6613 Mon Sep 17 00:00:00 2001
From: Drew Moseley <drew.moseley@northern.tech>
Date: Wed, 31 May 2017 11:57:06 -0400
Subject: [PATCH 5/6] u-boot-neo: Add CMD_PXE to udoo_neo_defconfig

Signed-off-by: Drew Moseley <drew.moseley@northern.tech>
---
 configs/udoo_neo_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/udoo_neo_defconfig b/configs/udoo_neo_defconfig
index 1fbf7e8..fb18043 100644
--- a/configs/udoo_neo_defconfig
+++ b/configs/udoo_neo_defconfig
@@ -25,3 +25,4 @@ CONFIG_CMD_TIME=y
 CONFIG_CMD_EXT4_WRITE=y
 CONFIG_OF_LIBFDT=y
 CONFIG_OF_LIBFDT_OVERLAY=y
+CONFIG_CMD_PXE=y
-- 
2.7.4

